package.json

```diff
@@ -4,6 +4,8 @@
   "description": "queue plugins",
   "main": "index.js",
   "scripts": {
+    "lint": "eslint smart-queue/",
```

Comment:
Until such a time we decide a folder structure (whether it be `/src` or lerna packages etc) we can point directly to smart-queue.